tests/foreman/ui/test_computeresource_gce.py

```diff
@@ -0,0 +1,158 @@
+"""Test for Compute Resource UI
+
+:Requirement: Computeresource GCE
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from fauxfactory import gen_string
+from nailgun import entities
+from pytest import skip
+from robottelo import ssh
+from robottelo.config import settings
+from robottelo.constants import (
+    COMPUTE_PROFILE_SMALL,
+    FOREMAN_PROVIDERS,
+    GCE_EXTERNAL_IP_DEFAULT,
+    GCE_MACHINE_TYPE_DEFAULT,
+    GCE_NETWORK_DEFAULT
+)
+from robottelo.decorators import bz_bug_is_open, fixture, setting_is_set, tier2
+
+
+if not setting_is_set('gce'):
+    skip('skipping tests due to missing gce settings', allow_module_level=True)
+
+
+class GCECertNotFoundError(Exception):
+    """An exception to raise when GCE Cert json is not available for creating GCE CR"""
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@fixture(scope='module')
+def module_loc():
+    return entities.Location().create()
+
+
+@fixture(scope='module')
+def module_gce_settings():
+    return dict(
+        project_id=settings.gce.project_id,
+        client_email=settings.gce.client_email,
+        cert_path=settings.gce.cert_path,
+        zone=settings.gce.zone
+    )
+
+
+@fixture(scope='module')
+def download_cert():
+    ssh.command('curl {0} -o {1}'.format(settings.gce.cert_url, settings.gce.cert_path))
+    if not ssh.command('[ -f {} ]'.format(settings.gce.cert_path)).return_code == 0:
+        raise GCECertNotFoundError("The GCE certificate {} is not found in satellite.".format(
+            settings.gce.cert_path))
```

Comment:
could you rewrite this to store the error output and include it in the exception?

Reply:
@JacobCallahan There wont be any stdout/stderr on checking if file exists or not with ```[ -f cert_path ]```. And -f does only checks for file existence and hence using this custom exception.

You want me to completely remove -f style file check ?